frontend/src/scenes/App.tsx

```diff
@@ -19,6 +19,20 @@ import { router } from 'kea-router'
 import { CommandPalette } from 'lib/components/CommandPalette'
 import { UpgradeModal } from './UpgradeModal'
 import { teamLogic } from './teamLogic'
+import posthog from 'posthog-js'
+import * as Sentry from '@sentry/browser'
+
+posthog.init((window as any).JS_POSTHOG_API_KEY || 'sTMFPsFhdP1Ssg', { api_host: (window as any).JS_POSTHOG_HOST })
+
+if ((window as any).SENTRY_DSN) {
+    Sentry.init({
+        dsn: (window as any).SENTRY_DSN,
```

Comment:
There's a ton of type hackery going on here, which makes this code hard to check - e.g. does Sentry.init accept undefined as dsn?

This can be all bypassed by modifying `frontend/src/globals.d.ts` instead of using the any-cast here.

Reply:
Done!